test/benchmark-crud/main.cpp

```diff
@@ -71,8 +71,8 @@ inline void erase(TableRef table, const std::vector<size_t> order)
 
 int main()
 {
-    const size_t target_size = 1100 * 1000L;
-    const int num_tables = 50;
+    const size_t target_size = 1100 * 100L;
```

Comment:
@finnschiermer do we lose a significant insight when looking at two levels rather than three? It increases the run-time of each iteration significantly when using another order of magnitude of rows.